wicketstuff-pdf-js-parent/wicketstuff-pdfjs/src/main/java/org/wicketstuff/pdfjs/res/wicketstuff-pdf.js

```diff
@@ -73,9 +89,116 @@
                         }
                     });
                     Wicket.Event.publish(WicketStuff.PDFJS.Topic.CURRENT_PAGE, pageNum, {"canvasId": config.canvasId});
+                    Wicket.Event.publish(WicketStuff.PDFJS.Topic.CURRENT_ZOOM, autoScale || scale.toFixed(2), {"canvasId": config.canvasId});
                 });
             }
 
+            function printDocument() {
+                pageRendering = true;
+                abortingPrinting = false;
+                var pages = pdfDoc.numPages;
+
+                var body = document.getElementsByTagName('body')[0];
+                body.setAttribute('pdf-js-printing', true);
+
+                var pc = document.createElement('div');
+                pc.id = 'pdf-js-print-container';
```

Comment:
I think using jQuery would be safer.
But I personally don't care about old browsers.
Don't spend your time on this!